extensions/MozChangeField/web/js/severity-s1-priority-p1.js

```diff
@@ -21,6 +21,7 @@ Bugzilla.SeverityS1PriorityP1 = class SeverityS1PriorityP1 {
   constructor() {
     this.priority = document.querySelector("#priority");
     this.severity = document.querySelector("#bug_severity");
+    if (!(this.priority && this.severity)) return;
```

Comment:
This and the change at line 35 fixes javascript errors that were preventing subsequent code from executing.